ngrinder-core/src/main/java/net/grinder/engine/agent/PropertyBuilder.java

```diff
@@ -190,6 +191,7 @@ public String buildJVMArgumentWithoutMemory() {
 		}
 
 		addParam(jvmArguments, properties.getProperty("grinder.param", ""));
+		addTargetHosts(jvmArguments);

```

Comment:
What this method does is identical with what the `addCustomDns()` method does.
You can access target hosts with `System.getProperty("ngrinder.etc.hosts")`.
It is not necessary to add this attribute.